src/components/common/BottomSheet/index.tsx

```diff
@@ -19,13 +19,14 @@ export const BottomSheet = memo(({ children, isOpen, setIsOpen }: Props) => {
     <Portal isOpen={isOpen}>
       <motion.div
         ref={bottomSheetRef}
-        className="absolute bottom-0 left-0 w-full bg-white shadow-[0px_-4px_10px_0px_rgba(0,0,0,0.1)] rounded-t-3xl p-4 overflow-hidden z-40 h-${sheetHeight}"
+        className="absolute bottom-0 left-0 right-0 m-auto max-w-[30rem] h-${sheetHeight} bg-white shadow-[0px_-4px_10px_0px_rgba(0,0,0,0.1)] rounded-t-3xl p-4 overflow-hidden z-40"
```

Comment:
벡틱으로 감싸야 동적으로 `sheetHeight` 반영가능할 거 같아요~!

```suggestion
        className={`absolute bottom-0 left-0 right-0 m-auto max-w-[30rem] h-${sheetHeight} bg-white shadow-[0px_-4px_10px_0px_rgba(0,0,0,0.1)] rounded-t-3xl p-4 overflow-hidden z-40`}
```